Revisions & additions to Model Versions (#3232)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/govern/model-contracts.md

---

:::info New functionality

> This functionality is new in v1.5 — if you have thoughts, weigh into the [GitHub discussion](https://github.com/dbt-labs/dbt-core/discussions/6726)!

Reviewer: "Weigh in" might be vague.
```suggestion
This functionality is new in v1.5 — if you have feedback, then participate in the [GitHub discussion](https://github.com/dbt-labs/dbt-core/discussions/6726)!
```

Author: 👍 replacing with ~"comment on"~ "participate in"
